frontend/app/src/components/defi/LoanInfo.vue

```diff
@@ -0,0 +1,387 @@
+                </div>
+                <div>
+                  <amount-display
+                    fiat-currency="USD"
+                    show-currency="ticker"
+                    :value="vault.liquidationPrice"
+                  ></amount-display>
+                </div>
+              </div>
+              <v-divider class="my-4"></v-divider>
+              <div class="d-flex justify-space-between">
+                <div>
+                  Minimum ratio
+                </div>
+                <div>
+                  {{ vault.liquidationRatio }}
+                </div>
+              </div>
+            </div>
+            <div>
+              <span class="loan-info__liquidation__liquidation-events__header">
+                Liquidation Events
+              </span>
+              <v-skeleton-loader
+                v-if="premium"
+                :loading="typeof vault.totalLiquidatedAmount === 'undefined'"
+                class="mx-auto pt-3"
+                max-width="450"
+                type="paragraph"
+              >
+                <div
+                  v-if="
+                    vault.totalLiquidatedAmount &&
+                    vault.totalLiquidatedAmount.gt(0)
+                  "
+                  class="loan-info__liquidation-events__content"
+                >
+                  <div
+                    class="loan-info__liquidation-events__content__liquidated-collateral mb-2"
+                  >
+                    <div
+                      class="d-flex justify-space-between font-weight-medium"
+                    >
+                      <div>
+                        Liquidated collateral
+                      </div>
+                      <div>
+                        <amount-display
+                          :value="vault.totalLiquidatedAmount"
+                          :asset="vault.collateralAsset"
+                        ></amount-display>
+                      </div>
+                    </div>
+                    <div class="d-flex justify-end">
+                      <div>
+                        <amount-display
+                          :value="vault.totalLiquidatedUsd"
+                          show-currency="ticker"
+                          fiat-currency="USD"
+                        ></amount-display>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="d-flex justify-space-between font-weight-medium">
+                    <div>
+                      Outstanding debt at liquidation
+                    </div>
+                    <div>
+                      <amount-display
+                        :value="vault.totalInterestOwed"
+                        asset="DAI"
+                      ></amount-display>
+                    </div>
+                  </div>
+                  <v-divider class="my-4"></v-divider>
+                  <div class="d-flex justify-space-between font-weight-medium">
```

Comment:
If this pattern is repeated a lot maybe we could extract a sub-component.

Reply:
Also here, we should evaluate if we have repeating patterns of template and if it makes to extract new components